msteams-platform/bots/how-to/conversations/command-bot-in-teams.md

```diff
@@ -0,0 +1,335 @@
+
+You can edit the manifest template file `templates\appPackage\manifest.template.json` to include:
+
+* The command title that user types in the message compose area to trigger the command.
+* The description for the command.
+
+  :::image type="content" source="../../../assets/images/command-bot-teams/commandbot-add-command-definition.png" alt-text="add a command definition in manifest code sample" lightbox="../../../assets/images/command-bot-teams/commandbot-add-command-definition.png":::
+
+<br>
+</details>
+
+<details>
+
+<summary><b>2. Respond with an Adaptive Card</b></summary>
+
+You can build your response data in text format, or perform the following steps to use an Adaptive Card to apply rich content in Teams:
+
+* Prepare your Adaptive Card content in a JSON file such as `myCard.json` under the `bot/adaptiveCards` folder. Following is a sample Adaptive Card with JSON payload:
+
+  ```JSON
+        {
+            "type": "AdaptiveCard",
+            "body": [
+            {
+                "type": "TextBlock",
+                "size": "Medium",
+                "weight": "Bolder",
+                "text": "Your Hello World Bot is Running"
+            },
+            {
+                "type": "TextBlock",
+                "text": "Congratulations! Your hello world bot is running. Click the documentation below to learn more about Bots and the Teams Toolkit.",
+                "wrap": true
+            }
+            ],
+            "actions": [
+            {
+                "type": "Action.OpenUrl",
+                "title": "Bot Framework Docs",
+                "url": "https://docs.microsoft.com/en-us/azure/bot-service/?view=azure-bot-service-4.0"
+            },
+            {
+                "type": "Action.OpenUrl",
+                "title": "Teams Toolkit Docs",
+                "url": "https://aka.ms/teamsfx-docs"
+            }
+            ],
+            "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+            "version": "1.4"
+        }
+  ```
+
+* Import your card content into your code file where your command handler exists: `import myCard from "./adaptiveCards/myCard.json"`.
+
+* In your `handleCommandReceived` API, use `MessageBuilder.attachAdaptiveCardWithoutData` or `MessageBuilder.attachAdaptiveCard` to build a bot message activity with the Adaptive Card and return the message `return MessageBuilder.attachAdaptiveCardWithoutData(myCard);`.
+
+For more information on how to build command and response using Adaptive Card with dynamic content, see this [section](#how-to-build-command-and-response-using-adaptive-card-with-dynamic-content).
+
+<br>
+
+</details>
+
+<details>
+
+<summary><b>3. Handle the command</b></summary>
+
+To handle the command, perform the following steps:
+
+1. Add a TypeScript or JavaScript file such as `xxxCommandHandler.ts` under `bot/src` to handle your bot command, and include the following sample code to get started:
+
+   ```TypeScript
```

Comment:
Can we update the code snippet for `3. Handle the command` to keep consistent with the readme?
https://github.com/OfficeDev/TeamsFx/tree/dev/templates/bot/ts/command-and-response#step-3-handle-the-command

Reply:
This has been updated.